puppet-client.html

```diff
@@ -60,18 +61,122 @@
                 border-color: #ffe070;
             }
             .error {
-                background-color: #fad7d4;
-                border-color: #f08a81;
+                background-color: #DB0F13;
```

Comment:
IMO this red is too bright and intense (in contrast to rest of our color palette). The previous one was specifically paired with the `.notice` color. If you change it I think you should change both (or at least check how they work together).